helm/reana/values.yaml

```diff
@@ -52,12 +52,12 @@ components:
     docs_url: http://docs.reana.io
     forum_url: https://forum.reana.io
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-ui:0.7.0
+    image: reanahub/reana-ui:0.7.2
   reana_db:
     enabled: true
   reana_server:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-server:0.7.1
+    image: reanahub/reana-server:0.7.3
     environment:
       REANA_MAX_CONCURRENT_BATCH_WORKFLOWS: 30
       REANA_USER_CONFIRMATION: true
```

Comment:
BTW it would be more precise to name the option `REANA_USER_EMAIL_CONFIRMATION`.  Dunno if it makes sense to change this at this stage... Since this affects only `reana` and `reana-server` repositories, we could consider this perhaps?

Reply:
I agree it is clearer, we can do it now since https://github.com/reanahub/reana-server/pull/336 isn't merged yet.

Reply:
Renamed everywhere.